conf/upf.json

```diff
@@ -69,6 +69,17 @@
     
     "": "p4rtc interface settings",
     "p4rtciface": {
+    "qci_to_priority_map": {
+        "qci_ngbr_5": 1,
+        "qci_ngbr_6": 6,
+        "qci_ngbr_7": 7,
+        "qci_ngbr_8": 8,
+        "qci_ngbr_9": 9,
+        "qci_ngbr_69": 1,
+        "qci_ngbr_70": 6,
+        "qci_ngbr_79": 7,
+        "qci_ngbr_80": 7
+    },
```

Comment:
is this specific to P4?

Reply:
its a mapping from qci to priority values. in p4rt we send this as scheduling priority to fast path. If it will be used by bess, we can move it to a common config also.

Reply:
1.  any specific reason for defining qci_ngbr_5 ,6,7 priority same as qci_ngbr_69,70,79 respectively (where as specs define unique priority level for every 5qi value)
2.  what about  GBR & delay critical gbr's  ?

Reply:
We should have a very similar slow path module so for the Tofino path you should map these priorities in the fast-path module. Remember that at the end of the day we want a single slow path supporting various fast path.  Then we keep the slow path fully 3GPP compliant